ftl/core/media-check.ftl

```diff
@@ -73,3 +73,7 @@ media-check-empty-trash = Empty Trash
 # button to move deleted files from the trash back into the media folder
 media-check-restore-trash = Restore Deleted
 media-check-check-media-action = Check Media
+# a tag for notes with missing media files (must not contain whitespace)
+media-check-missing-media-tag = missing-media
+# add a tag to notes with missing media
+media-check-add-tag = Add tag '{ media-check-missing-media-tag }'
```

Comment:
```suggestion
media-check-add-tag = Tag Missing
```

Reply:
Your approach does it make it more obvious to the user which tag they should look for, but we may be displaying a number of different buttons at once, and the buttons can end up getting truncated already, so brevity is probably the better option here. About to build a new rc so I'll push this through without waiting for your feedback, but feel free to make adjustments/discuss if you disagree.